save-orchestrator/src/test/kotlin/com/saveourtool/save/orchestrator/controller/agents/AgentsControllerTest.kt

```diff
@@ -196,11 +196,31 @@ class AgentsControllerTest {
     }
 
     private fun makeRequestToSaveLog(text: List<String>): WebTestClient.ResponseSpec {
-        val executionLogs = ExecutionLogs("agent", text)
+        val fileName = "agent.log"
+        val filePath = configProperties.executionLogs + File.separator + fileName
```

Comment:
Maybe create the file in temporary directory, since full path is not checked in the test?